cvat/settings/testing_rest.py

```diff
@@ -24,3 +24,11 @@
 HEALTH_CHECK = {
     'DISK_USAGE_MAX': 100,  # percent
 }
+
+# We don't really need to keep the cache for tests for a long time,
+# as this may cause some tests to fail due to the presence of data
+# in the cache when it is not expected.
+# We used to reset the cache every time we reset the data fixtures.
```

Comment:
Can we do an equivalent of that now?

The timeout-based approach seems iffy, since the behavior is different from the production configuration (and I don't think we can guarantee that it would take more than one second to start the next test).